migrations/d7_media_theplatform_mpx_source_field_storage.yml

```diff
@@ -1,6 +1,10 @@
 id: d7_media_theplatform_mpx_source_field_storage
 label: Source field for mpx objects
 migration_group: media_theplatform_mpx
+# This migration cannot be attached to the Drupal 7 tag as it uses the Empty
+# plugin, which doesn't define the required source_module key.
+#migration_tags:
+#  - 'Drupal 7'
```

Comment:
Is there a ticket for that? Can I help?

Reply:
I think this is a core issue. Empty literally means "no source" but core assumes all migrations from Drupal 7 have a source. I'm starting to think that perhaps we're overusing this tag in our migrations.